client/components/search/autocompleters/product-cat.js

```diff
@@ -0,0 +1,60 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+
+/**
+ * Internal dependencies
+ */
+import { computeSuggestionMatch } from './utils';
+import { stringifyQuery } from 'lib/nav-utils';
+
+/**
+ * A product categories completer.
+ * See https://github.com/WordPress/gutenberg/tree/master/packages/components/src/autocomplete#the-completer-interface
+ *
+ * @type {Completer}
+ */
+export default {
+	name: 'product_cats',
```

Comment:
how about just `categories`?

Reply:
I want to match the API/WP core names, so I think we should stick to `product_cats` (well, the core name is `product_cat`, but I pluralized `products` in the other autocompleter). There's also the possibility that we (or a 3rd party dev) might want a post-categories autocompleter, which would use `cats` or `categories`.

Reply:
😸 it is! I agree when we can stick to existing naming standards is probably best.